web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -114,6 +114,14 @@ export const SidebarCardsList = (props: Props): ReactElement => {
     <>
       <h2 className="h1-styling margin-top-0">{Config.dashboardDefaults.sidebarHeading}</h2>
       <hr className="margin-top-2 margin-bottom-3 bg-cool-lighter" aria-hidden={true} />
+      {props.topCards.length === 0 && props.bottomCards.length === 0 && (
```

Comment:
[boulder] I think we need to change the way we conditionally display this.

<img width="578" alt="Screenshot 2023-10-03 at 2 16 00 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22647707/1a928079-283e-4d33-8250-698da468f8aa">